### PR TITLE
Update DedicatedHost properties for 2020-09-30

### PR DESCRIPTION
**Pull request by dochung4**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [y] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [y] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [y] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by ArcturusZhang** [comment]
Hi @dochung4 could you please resolve the Avocado error?

**Comment by dochung4** [comment]
> Hi @dochung4 could you please resolve the Avocado error?

@ArcturusZhang , I tried to resolve the error, but it still fails. Do you know why, and is there an option to re-run the pipeline?

The error states that the example files that I added under 2020-09-30/examples are not referenced from the swagger file, but they are being referenced by compute.json under 2020-09-30. I double-checked and ensured that there are no typos.

**Comment by dochung4** [comment]
> /azp run

@ArcturusZhang , thank you for re-running the pipeline, but it failed again. Is it possible that the reason is that compute.json under 2020-09-30 is not being recognized as a Swagger file? This appears to be the case, since there are no references to it in readme.md. If so, would it be fine to ignore the failure?

**Comment by ArcturusZhang** [comment]
@dochung4 it is because 2020-09-30/compute.json is not referenced in the readme.md. Could you please introduce a new tag that contains the 2020-09-30/compute.json? Usually you need to duplicate the latest tag, rename it and replace the compute.json in it with this 2020-09-30/compute.json.

**Comment by dochung4** [comment]
> @dochung4 it is because 2020-09-30/compute.json is not referenced in the readme.md. Could you please introduce a new tag that contains the 2020-09-30/compute.json? Usually you need to duplicate the latest tag, rename it and replace the compute.json in it with this 2020-09-30/compute.json.

I tried that, and it fails with a different failure. I updated the PR to remove examples, since adding examples is not the main purpose of the PR. I will work on it separately.

**Comment by ArcturusZhang** [comment]
Now avocado has passed. We can first ignore the breaking change CI, because I suppose this is not changing it to a stronger constraint, instead, this PR introduces a weaker constraint and it should not be a breaking change.

Let me ask the owner of the breaking change CI

**Comment by dochung4** [comment]
> Now avocado has passed. We can first ignore the breaking change CI, because I suppose this is not changing it to a stronger constraint, instead, this PR introduces a weaker constraint and it should not be a breaking change.
> 
> Let me ask the owner of the breaking change CI

Yeah I am not sure why it shows up as a breaking change. Thank you for checking!

**Comment by ArcturusZhang** [comment]
Hi @dochung4 I confirmed with the owner of breaking change CI, it turns out that there is a bug in breaking change CI which makes it shows the wrong error message. 
The right error message for this should be `ConstraintIsWeaker` which is also a breaking change unfortunately... You could check this rule [here](https://github.com/Azure/openapi-diff/blob/master/docs/rules/1037.md)

Given that your PR is targeting the compute-09-01 branch, it is not very good to modify something in 2020-06-01 api-version, could you please split the changes for 2020-06-01 in another PR? We could deal the breaking changes in that PR.

**Comment by dochung4** [comment]
> Hi @dochung4 I confirmed with the owner of breaking change CI, it turns out that I had some misunderstanding about breaking changes and there is a bug in breaking change CI which makes it shows the wrong error message.
> The right error message for this should be `ConstraintIsWeaker` which is also a breaking change unfortunately... You could check this rule [here](https://github.com/Azure/openapi-diff/blob/master/docs/rules/1037.md)
> 
> Given that your PR is targeting the compute-09-01 branch, it is not very good to modify something in 2020-06-01 api-version, could you please split the changes for 2020-06-01 in another PR? We could deal the breaking changes in that PR.

Okay sure. Done.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/compute.json [hunk 1]
```diff
@@ -6739,8 +6739,8 @@
           "type": "integer",
           "format": "int32",
           "minimum": 1,
-          "maximum": 3,
-          "description": "Number of fault domains that the host group can span."
+          "maximum": 5,
```
Comment (johanste) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/compute.json:6742]: Unless we are confident that this value will never ever go up in the future, please remove the `maximum` attribute.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/compute.json [hunk 2]
```diff
@@ -6907,7 +6907,7 @@
           "type": "integer",
           "format": "int32",
           "minimum": 0,
-          "maximum": 2,
+          "maximum": 4,
```
Comment (johanste) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/compute.json:6910]: Unless we are confident that this value will never ever go up in the future, please remove the `maximum` attribute.

**Pull request closed without merging**